design/user_types.go

```diff
@@ -36,6 +42,8 @@ var UpdateWorkItemPayload = Type("UpdateWorkItemPayload", func() {
 var CreateWorkItemTypePayload = Type("CreateWorkItemTypePayload", func() {
 	Attribute("name", String, "Readable name of the type like Task, Issue, Bug, Epic etc.", func() {
 		Example("Epic")
+		Pattern("^[\\p{L}]+$")
```

Comment:
yes. WorkItem would be something like "system.userstory", "system.epic" --> <something>dot<something>. BUT WorkItemType would be something like "userstory", "epic".